src/main/kotlin/mods/eln/gridnode/GridSwitch.kt

```diff
@@ -286,13 +287,13 @@ class GridSwitchElement(node: TransparentNode, descriptor: TransparentNodeDescri
     override fun getWaila(): Map<String, String> {
         val info = mutableMapOf<String, String>()
         if (Eln.wailaEasyMode) {
-            info["Left"] = Utils.plotUIP(grida.voltage, grida.current)
-            info["Right"] = Utils.plotUIP(gridb.voltage, gridb.current)
-            info["Transfer"] = Utils.plotPower(transfer.power)
+            info[I18N.tr("Left")] = Utils.plotUIP(grida.voltage, grida.current)
```

Comment:
Nit: In Kotlin, you can just import it so that `tr()` is valid w/o `I18N.`  in front of it. This applies to all Kotlin files in this PR that are like this, I'm only going to note it once.

Reply:
In the next commit, I will replace all instances of "I18N.tr" with "tr". This will modify some of the content before the PR.